orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -51,12 +51,12 @@
   private final OutputFile file;
   private final Writer writer;
   private final VectorizedRowBatch batch;
-  private final OrcValueWriter<D> valueWriter;
+  private final OrcRowWriter<D> valueWriter;
```

Comment:
What is the purpose of changing `OrcValueWriter` to `OrcRowWriter`? It seems like this isn't really necessary for the purpose of this commit and causes quite a few changes. Is there value in doing this?

Reply:
Okay, sounds good to me.